public/assets/js/picks.js

```diff
@@ -134,12 +134,15 @@ function resetPage() {
     console.log("picksEntered = ");
     console.log(picksEntered);
 
-    $("#gm1pick").css("background-color", "#ffffff");
+    $("#gm1pick").css("background-color", "unset");
     $("#gm2pick").css("background-color", "#dddddd");
-    $("#gm3pick").css("background-color", "#ffffff");
+    $("#gm3pick").css("background-color", "unset");
     $("#gm4pick").css("background-color", "#dddddd");
     for(let i = 1; i<5; i++) {
-        $(`#gm${i}pick`).css("text-decoration", "none");
+        
+        $(`#gm${i}pick`).css("font-weight", "bold");
+        // $(`#gm${i}pick`).css("text-decoration", "none");
```

Comment:
I left the original one commented out here in case what I did screws something up.